### PR TITLE
WIP 27 document dev setup fix #27

### LINKED ISSUE #27: document dev setup

### PR DESCRIPTION
**Pull request by micahstubbs**
A WIP PR so that @enjalot can easily suggest changes in the comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,3 +10,31 @@ This is an express endpoint which calls ElasticSearch and returns the results.
...
+### setup search index backend
+
+Follow the steps in the [Setup Elasticsearch & Index some Gists](https://github.com/enjalot/blockbuilder-search-index#setup-elasticsearch--index-some-gists) section of the **blockbuilder-search-index** project
+
+These steps guide us through setting up, installing, and running a local Elasticsearch instance that we can populate with blocks.  The blocks data is retrieved from Github and stored as static json files, before being loaded into our Elasticsearch index.  Once our search index is up, running, and populated, we're reading to query it with a web ui. 
+
+### connect local blockbuilder with local blockbuilder-search
+
+```
+// npm link blockbuilder-search
```
Comment (enjalot) [inline README.md]: We need 2 steps here:
cd blockbuilder-search
npm link
cd ../blockbuilder
npm link blockbuilder-search

I don't usually use // when documenting terminal commands. should be able to do ```bash

File: README.md [hunk 2]
```diff
@@ -10,3 +10,31 @@ This is an express endpoint which calls ElasticSearch and returns the results.
...
+
+### setup blockbuilder dev environment
+
+Follow the [full dev setup guide](https://github.com/enjalot/blockbuilder/wiki/Development#development) at the **blockbuilder** code-editor project
+
+Once you've done that, verify that it worked by opening a web browser and visiting local blockbuilder at [http://[::]:8889](http://[::]:8889)
+
+### setup search index backend
+
+Follow the steps in the [Setup Elasticsearch & Index some Gists](https://github.com/enjalot/blockbuilder-search-index#setup-elasticsearch--index-some-gists) section of the **blockbuilder-search-index** project
```
Comment (enjalot) [inline README.md]: should we link to the specific elasticsearch version? the readme doesn't say anything about installing elasticsearch
https://www.elastic.co/downloads/past-releases/elasticsearch-2-3-4

Comment (micahstubbs) [reply README.md]: yes we should.  after this PR https://github.com/enjalot/blockbuilder-search-index/pull/38, the **blockbuilder-search-index**  **README.md** should link to that specific version of Elasticsearch.  

line https://github.com/enjalot/blockbuilder-search-index/pull/38/files#diff-04c6e90faac2675aa89e2176d2eec7d8R120

File: README.md (hunk 1 above)
Comment (micahstubbs) [reply README.md]: that's right `# bash comment`  

got lazy there 😉

**Comment by micahstubbs** [comment]
👍

**Pull request merged**